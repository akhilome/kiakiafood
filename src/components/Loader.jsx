import React from 'react';

const Loader = () => (
  <div className="loader">
    <img src="https://i.imgur.com/S39IaLt.gif" alt="loading gif" />
  </div>
);

export default Loader;
