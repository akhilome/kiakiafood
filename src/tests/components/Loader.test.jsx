import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../../components/Loader';

describe('<Loader />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Loader />);
    expect(wrapper).toMatchSnapshot();
  });
});
