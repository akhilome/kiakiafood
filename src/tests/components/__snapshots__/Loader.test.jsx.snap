// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should render correctly 1`] = `
<div
  className="loader"
>
  <img
    alt="loading gif"
    src="https://i.imgur.com/S39IaLt.gif"
  />
</div>
`;
